src/main/java/com/to/ordermanagementservice/dto/OrderItemDetails.java

```diff
@@ -1,15 +1,22 @@
 package com.to.ordermanagementservice.dto;
 
 import java.math.BigDecimal;
+import java.time.OffsetDateTime;
 
 public class OrderItemDetails {
 
     private Integer orderItemId;
 
     private Integer productId;
 
+    private Integer id;
```

Comment:
why do you nee both `orderItemId` and `id` in this class?

Reply:
on task write 
Extend the GetAllOrders REST API to return all orders in the system:

Order entity
id
customerId
createdAt
updatedAt
orderItems[]
id
productId
quantity
price
createdAt
updatedAt

Reply:
the "orderItemId" is not in the list